spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -378,15 +398,26 @@
           stub_request_proofing_results_with_responses({})
         end
 
-        it_behaves_like('enrollment encountering an exception', reason: 'Bad response structure')
+        it_behaves_like(
+          'enrollment encountering an exception',
+          reason: 'Bad response structure',
+          response_json: UspsInPersonProofing::
+          Mock::Fixtures.
+          request_passed_proofing_unsupported_status_results_response,
```

Comment:
nit: preference for this formatting. Same with the changes below

```suggestion
          response_json: UspsInPersonProofing::Mock::Fixtures.
            request_passed_proofing_unsupported_status_results_response,
```

Reply:
Thanks!  I made the formatting changes and I'm going to merge the PR now.  I agree that it would be good to get a rubocop rule to enforce the style suggestions.